src/uisMultipleDirective.js

```diff
@@ -0,0 +1,393 @@
+uis.directive('uisMultiple', ['uiSelectMinErr','$timeout', function(uiSelectMinErr, $timeout) {
```

Comment:
The prefix `uis` is already used on some filenames/directives of this library to keep some names shorter, but we could also consider `uiSelectMultiple`and `uiSelectSingle` for this new directives if if thats cleaner.